### PR TITLE
Changelog: Updates changelog with more feature details

### PR DESCRIPTION
**Pull request by torkelo**
Because in 7.0 we have some big features that feel like they would get lost in the long list of issues maybe something like this is a good idea?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,42 @@
...
+
+## 7.0 Feature highlights
+
+### Data transformations
+
+Not just visualizing data from anywhere, in Grafana 7 you can transform it too. By chaining a simple set of point and click transformations users will be able join, pivot, filter, re-name and calculate to get the results they need. Perfect for operations across queries or data sources missing essential data transformations.
+
+Data transformations will provide a common set of data operations that were previously duplicated as custom features in many panels or data sources but are now an integral part of the Grafana data processing pipeline and something all data sources and panels can take advantage of.
+
+In Grafana 7.0 we have a shared data model for both time series and table data. We refer to table columns as fields in both the UI and docs. A time series is simply a table
```
Comment (dprokop) [inline CHANGELOG.md:18]: Maybe link to https://github.com/grafana/grafana/blob/master/docs/sources/plugins/developing/dataframe.md?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,42 @@
...
+with two fields (time & value).
+
+**Transformations shipping in 7.0**
+
+- **Reduce**: Reduce many rows / data points to a single value
+- **Filter by name**: Filter fields by name or regex
+- **Filter by refId**: Filter by query letter
+- **Organize fields**: Re-order, rename and hide fields.
```
Comment (dprokop) [inline CHANGELOG.md:26]:
```suggestion
- **Organize fields**: Reorder, rename and hide fields.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,42 @@
...
+- **Labels for fields**: Transform time series with labels into a table where labels get's converted to fields and the result is joined by time
```
Comment (dprokop) [inline CHANGELOG.md:27]:
```suggestion
- **Labels to fields**: Transform time series with labels into a table where labels get's converted to fields and the result is joined by time
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,42 @@
...
+- **Join by field**: Join many result sets (series) together using for example the time field. Useful for transforming time series into a table with a shared time column and where each series get it's own column.
+- **Add field from calculation**: This is super powerful transformation that allows you perform many different types of math operations and add the result as a new field. Examples:
```
Comment (dprokop) [inline CHANGELOG.md:29]:
```suggestion
- **Add field from calculation**: This is a powerful transformation that allows you perform many different types of math operations and add the result as a new field. Examples:
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,42 @@
...
+  - Calculate the difference between two series or fields and add the result to a new field
+  - Multiply one field with another another and add the result to a new field
+
+### New panel edit experiance
+
+In Grafana 7 we have completely updated the UI for editing panels. The first visible change is that we have seperated panel display settings to a right hand side pane that you can collapse or expand depending on what your focus is on. With this change we also introducing our new unified option model & UI for defining data configuration & display options. This unified data configuration system powers a consistent UI for setting data options across visualizations as well as making all data display settings data driven and overridable.
```
Comment (dprokop) [inline CHANGELOG.md]:
```suggestion
In Grafana 7 we have redesigned the UI for editing panels. The major change is that we have separated panel display settings to a right hand side pane that you can collapse or expand depending on what your focus is on. With this change we also introducing our new unified option model & UI for defining data configuration & display options. This unified data configuration system powers a consistent UI for setting data options across visualisations as well as making all data display settings data driven and overridable.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,42 @@
...
+
+This new option architecture and UI will make all panels have a consistent set of options and behavior for attributes like `unit`, `min`, `max`, `thresholds`, `links`, `decimals`. Not only that but all these options will share a consistent UI for specifying override rules and is extensible for custom panel specific options.
+
+We have yet to migrate all core panels to this new architecute so in 7.0 there will sadly be some big inconsistencies in the UI between panels. Hopefully this will be fixed soon in future releases as we update all the core panels and help the community update the community panel plugins.
+
+### New tabel panel
+
+Grafana 7.0 comes with a new table panel (and deprecates the old one). This new table panel supports horizontal scrolling, column resize. And paired with the new `Organize fields` transformation detailed below you can re-order, hide & rename columns. This new panel also supports new cell display modes, like showing a bar gauge inside a cell.
```
Comment (dprokop) [inline CHANGELOG.md]:
```suggestion
Grafana 7.0 comes with a new table panel (and deprecates the old one). This new table panel supports horizontal scrolling and column resize. Paired with the new `Organise fields` transformation detailed below you can reorder, hide & rename columns. This new panel also supports new cell display modes, like showing a bar gauge inside a cell.
```

**Review by dprokop — COMMENTED** [review]
Missing the Panel inspect highlight

**Comment by torkelo** [comment]
thanks for the edits! yea missing Inspect, timezone support.

File: CHANGELOG.md (hunk 6 above)
Comment (torkelo) [reply CHANGELOG.md]: it's named Organize in the app

Comment (dprokop) [reply CHANGELOG.md]: ah yeah, tru

**Comment by torkelo** [comment]
They cover the buttons almost completely :( 

And I am not getting a tooltip for currently open tab, was that intentional?  

Can we use native tooltips for now?

**Comment by dprokop** [comment]
@torkelo comment on wrong PR... :)

**Pull request merged**